__tests__/server/listen.spec.js

```diff
@@ -113,24 +113,22 @@ describe('server listen', () => {
       expect(cb).toHaveBeenCalledWith(null, { port: '8998' });
     });
 
-    it('passes the error to the callback', (done) => {
+    it('passes the error to the callback', async () => {
       process.env.HTTP_PORT = 8998;
       const listenError = new Error('port taken or such');
       http.mock.listenError = listenError;
-      listenHttp(app, (err, serverInfo) => {
+      await listenHttp(app, (err, serverInfo) => {
```

Comment:
Should add `expect.assertions(2);`

Reply:
you only need expect.assertions for conditional test